pbs_light_test.go

```diff
@@ -623,3 +625,59 @@ func (validator *testValidator) Schema(name openrtb_ext.BidderName) string {
 		return "{\"appnexus\":false}"
 	}
 }
+
+// Test the viper setup
+func TestViperInit(t *testing.T) {
+	CompareStrings(t, "Viper error: external_url expected to be %s, found %s", "http://localhost:8000", viper.Get("external_url").(string))
+	CompareStrings(t, "Viper error: adapters.pulsepoint.endpoint expected to be %s, found %s", "http://bid.contextweb.com/header/s/ortb/prebid-s2s", viper.Get("adapters.pulsepoint.endpoint").(string))
+}
+
+func TestViperEnv(t *testing.T) {
+	// Record environment
+	port, set := os.LookupEnv("PBS_PORT")
```

Comment:
Could minimize duplicate code with a `func clearEnv(key string) (reseter func())`

I'm not sure how the Go syntax works on this... but you _may_ be able to `defer (clearEnv("PBS_PORT"))`. If that doesn't work, you can still reduce these 5 repeated lines to 2.